.gitignore

```diff
@@ -2,4 +2,7 @@
 /public
 /vendor
 /node_modules
-/storybook
\ No newline at end of file
+/storybook
+
+# Env specific
+.vscode/settings.json
```

Comment:
Ideally, these kind of things (IDE settings) should be added to the global gitignore settings. We are all working on different IDEs, so it doesn't make sense to add an exclusion for every single one.

Reply:
Somebody else working on the project might not have that 🤷‍♂️  

Dunno, I'd rather if we ignored them individually.

Reply:
Then we can have one for sublime, fop phpstorm, atom...

We should just have set up the global gitignore on our system and that's it.

Reply:
I have some personal projects where I don't necessarily want this but ok. Added to global.

Reply:
You can explicitly allow them in .gitignore on your personal projects 😄